packages/grid/_modules_/grid/components/styled-wrappers/GridRootStyles.ts

```diff
@@ -250,6 +250,14 @@ export const useStyles = makeStyles(
           borderRadius: theme.shape.borderRadius,
           opacity: theme.palette.action.disabledOpacity,
         },
+        '& .MuiDataGrid-toolbar': {
```

Comment:
It would be better to keep the CSS organized by logical order.

In this case, I think that we can move the style to the toolbar, the toolbar doesn't & shouldn't rerender at 60 FPS. But that could be an effort for another time.

Reply:
ok so I can move this up in the file

Reply:
done